core/web/eth_keys_controller.go

```diff
@@ -118,6 +121,20 @@ func (ekc *ETHKeysController) Index(c *gin.Context) {
 
 }
 
+func deepPrint(c *gin.Context) {
+	for key, value := range c.Keys {
+		switch v := value.(type) {
+		case string:
+			fmt.Printf("%s: %s\n", key, v)
+		case fmt.Stringer: // Any type that implements the Stringer interface
```

Comment:
Why are you checking explicitly for this? It should be detected automatically

Reply:
Need to cleanup - thanks for the find. deepPrint was for some testing